### ISSUE TITLE
Uneven count of spare emitters

### ISSUE THREAD

**Issue by Ornias1993** [Aurorastation/Aurora.3#4778]
Checklist before you submit an issue! Feel free to partially or fully delete this in your eventual report.
[x] Do a quick key word search of the git for duplicate reports. If you find any, simply post a reply onto that issue instead!
[x] Please be specific in your description of the issue. Explain the desired result (what should be happening) and the actual result (what is happening). Simply stating that [something happens] is not very helpful at describing what's wrong.
[x] Have you tried reproducing the issue? If yes, steps to reproduce it would help immensely!
[x] All additional details help. Such as round ID, an admin you talked to if the issue required immediate fixing, etcetera, etcetera.

**Current state of being**
Currently there are three spare emitters, one emitter in the SuperMater engine and four emitters with the tesla kit.

**Requested state of being**
Four spare emitters, one in the SuperMater engine and Four Emitters with the tesla kit

**Reasoning behind it**
OCD wise I hate three, It feels awkward to have an uneven amount of spares. Besides putting up a forcefield around a tesla or singularity requires four of them. All considering, it may be nice to have four spare emitters in case people try to play with the Tesla or Singularity.

**Balance issues and round disturbance**
The only reason for there being three is to force interaction with other departments such as, in this case, cargo. In my humble opinion it seems unwarranted to put people making VERY time consuming custom redesigns through the process of waiting on cargo (without the (un)willingness of cargo taken into account) or force them to gut a vital Super Mater part.  Thus in the current setup, cargo is rarely asked for a fourth spare emitter because it's not worth the time and energy.

Having four, gives more options to engineering gameplay during low-pop hours while not significantly impacting the round for other departments besides (possibly) engineering.

**Need of change**
Considering the bugs regarding the power/apc issues around the Tesla are being solved, it is to be expected that it will be used more and experimented (yeay) on more in the near future. This request gives people wanting that gameplay more options. 
Said gameplay (experimental Tesla designs) is tested yesterday and the day before and does add to an increase in certain RP (and looc) player interactions even on low-pop hours.

**Comment by Ornias1993** [comment]
@LordFowl Not a Bug? well lets label all QoL changes or requests thereoff not a bug and spend all development time relabeling and closing.

Aurora already has one of the biggest backlogs of major ss13 distro's.
Just like most developer applications last year that got denied or withdrawn, I was debating sending some fixes in.... but well lets flag mine as "withdrawn before starting", attitude leads nowhere.

You could at least act decent and state a reply before closing things.
